src/Umbraco.Core/PropertyEditors/ValueConverters/TagsValueConverter.cs

```diff
@@ -12,69 +10,42 @@ namespace Umbraco.Core.PropertyEditors.ValueConverters
     [DefaultPropertyValueConverter]
     public class TagsValueConverter : PropertyValueConverterBase
     {
-        private readonly IDataTypeService _dataTypeService;
+        public TagsValueConverter()
+        { }
 
+        [Obsolete("This constructor isn't required anymore, because we don't need services to lookup the data type configuration.")]
         public TagsValueConverter(IDataTypeService dataTypeService)
-        {
-            _dataTypeService = dataTypeService ?? throw new ArgumentNullException(nameof(dataTypeService));
-        }
+        { }
```

Comment:
I've obsoleted the original constructor, so it's clear to not use this one (although it doesn't really matter, as it doesn't do anything with the parameter) and added a parameterless one that can be used instead.